javalanglib/src/main/scala/java/lang/Iterable.scala

```diff
@@ -13,7 +13,14 @@
 package java.lang
 
 import java.util.Iterator
+import java.util.function.Consumer
 
 trait Iterable[T] {
   def iterator(): Iterator[T]
+
+  def forEach(action: Consumer[T]): Unit = {
+    val iter = iterator()
+    while (iter.hasNext)
```

Comment:
Our coding style asks that we always use `()` when calling Java methods. So this should be `.hasNext()`.